server/app/views/questiontypes/StaticTextQuestionFragment.html

```diff
@@ -1,4 +1,4 @@
-<div th:fragment="static-text(question)">
+<div th:fragment="static-text(question)" data-testid="staticQuestionRoot">
```

Comment:
_very_ nit but should this be `questionRoot` to match the others?

Reply:
ohh it's cause there's two questions on the page, isn't it?

Reply:
yep I had to give it a different Id because there are two questions